tests/fixtures/expected_tests_results.py

```diff
@@ -0,0 +1,208 @@
+EXPECTED_DIFF = """{
```

Comment:
it's better to store strings and diffs in different places (so you'll have `expected_diffs` (or just `diffs`) and `expected_dicts` (or just `dicts`))